src/finn/util/visualization.py

```diff
@@ -36,7 +36,29 @@ def showSrc(what):
     print("".join(inspect.getsourcelines(what)[0]))
 
 
-def showInNetron(model_filename):
-    netron.start(model_filename, address=("0.0.0.0", 8081))
-    localhost_url = os.getenv("LOCALHOST_URL", default="localhost")
-    return IFrame(src="http://%s:8081/" % localhost_url, width="100%", height=400)
+def showInNetron(model_filename: str, localhost_url: str = None, port: int = None):
+    """Shows a ONNX model file in the Jupyter Notebook using Netron.
+
+    :param model_filename: The path to the ONNX model file.
+    :type model_filename: str
+
+    :param localhost_url: The IP address used by the Jupyter IFrame to show the model.
+     Defaults to localhost.
+    :type localhost_url: str, optional
+
+    :param port: The port number used by Netron and the Jupyter IFrame to show
+     the ONNX model.  Defaults to 8081.
+    :type port: int, optional
+
+    :return: The IFrame displaying the ONNX model.
+    :rtype: IPython.lib.display.IFrame
+    """
+    if port is None:
+        try:
+            port = int(os.environ["NETRON_PORT"])
+        except KeyError:
+            port = 8081
```

Comment:
You can possibly re-write this to:
```
port = port or int(os.getenv("NETRON_PORT",  "8081"))
```
And maybe catch an int conversion exception instead.

Reply:
Thanks for the suggestion, that's a really good idea! I refactored the code accordingly and did the same adjustment for the `localhost_url` argument.